openhands/runtime/file_viewer_server.py

```diff
@@ -22,12 +22,12 @@ def create_app() -> FastAPI:
     )
 
     @app.get('/')
-    async def root():
+    async def root() -> Dict[str, str]:
```

Comment:
```suggestion
    async def root() -> dict[str, str]:
```

Prefer build in types rather than those from `typing`

Reply:
Same from `Tuple`